k8s/operator/templates/deployment.yaml

```diff
@@ -105,3 +105,82 @@ spec:
       tolerations:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+{{- end }}
+---
+{{- if has "git-operator" .Values.sdp.operators }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "sdp-operator.fullname" . }}-git
+  namespace: {{ .Release.Namespace }}
+  labels:
+  {{- include "sdp-operator.labels" . | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      {{- include "sdp-operator.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "sdp-operator.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.sdp.gitOperator.image.pullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      serviceAccountName: {{ include "sdp-operator.serviceAccountName" . }}-git
+      volumes:
+        - name: github-deployment-key
+          items:
+            - key: github-deployment-key
+              path: deployment.key
```

Comment:
I think this needs to be `/opt/git-operator/k8s/deployment.key` based on the value set for `GITHUB_DEPLOYMENT_KEY_PATH`

Reply:
https://github.com/appgate/sdp-operator/blob/995d17756d06db894bc93efe27841b5bf1e2558b/k8s/operator/templates/deployment.yaml#L146 <- It's mounted here